src/components/dds/Notification/Notification.tsx

```diff
@@ -0,0 +1,134 @@
+     * */
+    messageBody: string;
+    /**
+     * Time stamp for the notification
+     * */
+    timeStamp: string;
+    /**
+     * Primary action text for the notification
+     * */
+    primaryActionText: string;
+    /**
+     * Secondary action text for the notification
+     * */
+    secondaryActionText: string;
+    /**
+     * Update the number of Notifications visible on the Demo screen (Max of 3)
+     * */
+    notificationCount: number;
+    /**
+     * Whether the close icon should be visible
+     * */
+    closeIcon: boolean;
+    /**
+     * Icon name for the notification
+     * */
+    titleIcon: string;
+    /**
+     * Icon type for the notification
+     * */
+    titleIconType: string;
+    /**
+     * Callback function to be executed when primary action button is clicked.
+     * */
+    primaryAction: () => void;
+    /**
+     * Callback function to be executed when secondary action button is clicked.
+     * */
+    secondaryAction: () => void;
+}
+
+/**
+ * React implementation of DDS notification component
+ */
+export function Notification({
+    className,
+    title,
+    messageBody,
+    timeStamp,
+    primaryActionText,
+    secondaryActionText,
+    notificationCount,
+    closeIcon,
+    titleIcon,
+    titleIconType,
+    primaryAction,
+    secondaryAction,
+    style,
+    ...rest
+}: NotificationProps) {
+    const [visible, setVisible] = useState(true);
+
+    const StyledCloseButton = styled(Button)`
+        padding: 0;
+        font-size: 1em;
+        line-height: 1;
+        && > i {
+            margin-right: 0;
+            font-size: 1em;
+    `;
+
+    return (
+        <div
+            className={classNames(getRootClassNames(className), componentClasses)}
+            style={style}
+            {...rest}
+            data-dds="notification"
+        >
+            <div
+                className={classNames(componentClasses.base, visible && componentClasses.show)}
+                role="alert"
+                aria-live="polite"
+                aria-atomic="true"
+                aria-hidden={visible}
+            >
+                <Icon name={titleIcon} type="font-icon" className={componentClasses.icon} />
+                <div className={componentClasses.body}>
+                    <div className={componentClasses.header}>
+                        <span className={componentClasses.headerTitle}>{title}</span>
+                        {closeIcon && (
+                            <StyledCloseButton
+                                variant="tertiary"
+                                className={classNames(componentClasses.button.base, componentClasses.closeButton)}
+                                onClick={() => setVisible(false)}
+                                size="sm"
+                                iconOnly
+                            >
+                                <Icon name="close-x" type="font-icon" className={componentClasses.icon} />
```

Comment:
Optional:

Probably it will work as well:
```suggestion
                                <Icon name="close-x" className={componentClasses.icon} />
```

Reply:
done: https://github.com/cloudify-cosmo/cloudify-ui-components/pull/228/commits/67fc8954618e0dafc3fbde776ab900ae3af2c3df